fighters/packun/src/acmd/other.rs

```diff
@@ -156,31 +222,79 @@ unsafe fn appeal_hi_sound(fighter: &mut L2CAgentBase) {
     }
 }
 
-#[acmd_script( agent = "packun", script = "game_appealsl", category = ACMD_GAME , low_priority)]
-unsafe fn appeal_s_l_game(fighter: &mut L2CAgentBase) {
+#[acmd_script( agent = "packun", scripts = ["game_appealsl", "game_appealsr"] , category = ACMD_GAME , low_priority)]
```

Comment:
repeat my review for the other taunts

Reply:
this is so ugly I genuinely would rather you handle it via OPFF

Reply:
I think normally this would be done in status code and the resulting plant transformation form would just be passed in via workmodule

Reply:
I just don't think ACMD should be used like this, it doesn't feel like good practice
do an OPFF, do a taunt status and motion check, set a flag to true on frame 1 of ACMD and if the button is off before frame X turn off the flag.